bigbluebutton-html5/imports/ui/components/external-video-player/component.jsx

```diff
@@ -151,8 +198,16 @@ class VideoPlayer extends Component {
         logger.debug({ logCode: 'external_video_client_stop' }, 'Stop external video');
       });
 
+      onMessage('presenterReady', (data) => {
+        logger.debug({ logCode: 'external_video_presenter_ready' }, 'Presenter is ready to sync');
+
+        if (!this.state.hasPlayedBefore) {
+          this.setState({playing: true});
+        }
+      });
+
       onMessage('playerUpdate', (data) => {
-        if (!this.player) {
+        if (!this.player || !this.state.hasPlayedBefore) {
```

Comment:
I just noticed all of the `this.state.X` usage. They should be using destructuring.
```
const {
  X,
} = this.state;
```

Reply:
Addressed this in the newest commits